ra_aid/agents/ciayn_agent.py

```
@@ -1,6 +1,9 @@
-import inspect
+import re
 from dataclasses import dataclass
 from typing import Dict, Any, Generator, List, Optional, Union
+from typing import Dict, Any, Generator, List, Optional, Union
```

Ruff (0.8.2)

Import from `collections.abc` instead: `Generator`

Import from `collections.abc`

(UP035)